tests/fixtures/dynamodb_fixtures.py

```diff
@@ -0,0 +1,94 @@
+import os
+from typing import Any, Dict, Generator, List
+from uuid import uuid4
+
+import pytest
+from sqlalchemy.orm import Session
+
+from fides.api.ctl.sql_models import Dataset as CtlDataset
+from fides.api.ops.models.connectionconfig import (
+    AccessLevel,
+    ConnectionConfig,
+    ConnectionType,
+)
+from fides.api.ops.models.datasetconfig import DatasetConfig
+from fides.api.ops.schemas.connection_configuration.connection_secrets_dynamodb import (
+    DynamoDBSchema,
+)
+
+from .application_fixtures import integration_secrets
+
+
+@pytest.fixture(scope="function")
+def dynamodb_connection_config_without_secrets(db: Session) -> Generator:
+    """
+    Returns a DynamoDB ConnectionConfig without secrets
+    attached that is safe to use in any test.
+    """
+    connection_config = ConnectionConfig.create(
+        db=db,
+        data={
+            "name": str(uuid4()),
+            "key": "my_dynamodb_config",
+            "connection_type": ConnectionType.dynamodb,
+            "access": AccessLevel.write,
+            "secrets": integration_secrets["dynamodb_example"],
+        },
+    )
+    yield connection_config
+    connection_config.delete(db)
+
+
+@pytest.fixture(scope="function")
+def dynamodb_connection_config(
+    db: Any,  # type: ignore
+    integration_config: Dict[str, str],
+    dynamodb_connection_config_without_secrets: ConnectionConfig,
+) -> Generator:
+    """
+    Returns a DynamoDB ConectionConfig with secrets attached if secrets are present
+    in the configuration.
+    """
+    dynamodb_connection_config = dynamodb_connection_config_without_secrets
+    region = integration_config.get("aws", {}).get("region") or os.environ.get(
+        "DYNAMODB_REGION"
+    )
+    aws_access_key_id = integration_config.get("aws", {}).get(
```

Comment:
These fixtures fail to load locally, since the config dict is referenced as `aws` here and `dynamodb_example` in the [`integration_test_config.toml` file](https://github.com/ethyca/fides/pull/2998/files#diff-de8140bfa2cfbae36f4b1d426dce530a9a1b3aaa1f5bc9f06a3e3daab6e5b665R62)

Reply:
Great catch, thank you!

Reply:
Fixed in https://github.com/ethyca/fides/pull/2998/commits/6cbb1f664fa12be058b1ad1bfdf0c6db5f3b0f17